src/pipelines/sass.rs

```diff
@@ -81,16 +99,35 @@ pub struct SassOutput {
     /// The ID of this pipeline.
     pub id: usize,
     /// Data on the finalized output file.
-    pub file: HashedFileOutput,
+    pub css_ref: CssRef,
+}
+
+/// The resulting CSS of the SASS/SCSS compilation.
+/// Either a hashed file reference (default) or a String (for `data-inline`)
+pub enum CssRef {
+    Inline(String),
+    File(HashedFileOutput),
```

Comment:
Solid! Good idea to model this as an enum. Couple small items:
- Rust doc strings should have a blank line after the initial sentence describing the type. 
- However, that second line (`Either a hashed ...`), let's actually remove that and instead add a single line doc string above each of the variants. I think that would be preferable.

Thanks!

Reply:
Thanks;

I've just split the lines.